### PR TITLE
#547 Implemented tests for length of class

### PR DESCRIPTION
**Pull request by nick318**
Implemented tests for `length of` class for #547 issue

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#597](https://github.com/yegor256/cactoos/issues/597) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#597](https://github.com/yegor256/cactoos/issues/597) is assigned to @neonailol[/z](https://www.0crat.com/u/neonailol). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/test/java/org/cactoos/io/LengthOfTest.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                new InputOf("test buffer1"),
+                1
+            ).intValue(),
+            Matchers.equalTo(12)
+        );
+    }
+
+    // @checkstyle RegexpMultilineCheck (5 lines)
+    /**
+     * @todo #547:30m/DEV Fix length of for buffer of 0
```
Comment (neonailol) [inline src/test/java/org/cactoos/io/LengthOfTest.java]: please correct spelling in this todo

Comment (nick318) [reply src/test/java/org/cactoos/io/LengthOfTest.java]: @neonailol done

**Comment by neonailol** [comment]
@rultor good to merge

**Comment by nick318** [comment]
@yegor256 ping

**Comment by nick318** [comment]
@yegor256 ping2

**Comment by neonailol** [comment]
@yegor256 please take a look at this

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @neonailol[/z](https://www.0crat.com/u/neonailol), total is [+405](https://www.0crat.com/u/neonailol)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+9440](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#597](https://github.com/yegor256/cactoos/issues/597) is now out of scope

**Pull request merged**